static/scss/answers/common/mixins.scss

```diff
@@ -84,6 +84,112 @@
   {
     margin-bottom: var(--yxt-base-spacing);
   }
+
+  s
+  {
+    text-decoration: line-through;
+  }
+
+  sup
+  {
+    vertical-align: super;
+    font-size: smaller;
+  }
+
+  sub 
+  {
+    vertical-align: sub;
+    font-size: smaller;
+  }
+
+  code 
+  {
+    font-family: monospace;
+    font-size: smaller;
+  }
+
+  pre 
+  {
+    display: block;
+    font-family: monospace;
+    white-space: pre;
+    margin: 1em 0px;
+  }
+
+  blockquote 
+  {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: var(--yxt-base-spacing);
+    margin-inline-end: var(--yxt-base-spacing);
+  }
+
+  h1 
+  {
+    display: block;
+    font-size: var(--yxt-font-size-xlg);
+    margin-block-start: .33em;
+    margin-block-end: .33em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    font-weight: bold;
+  }
+
+  h2 
+  {
+    display: block;
+    font-size: var(--yxt-font-size-lg);
+    margin-block-start: .33em;
```

Comment:
do we want to make a mixin for the margin?

Reply:
sure, I can do that